src/renderer/components/ft-video-player/ft-video-player.js

```diff
@@ -251,6 +251,26 @@ export default Vue.extend({
           this.player.removeChild('BigPlayButton')
         }
 
+        // Makes the playback rate menu focus the current item on mouse hover
+        // or the closest item if the playback rate is between two items
+        // which is likely to be the case when the playback rate is changed by scrolling
+        const playbackRateMenuButton = this.player.controlBar.getChild('playbackRateMenuButton')
+        playbackRateMenuButton.on(playbackRateMenuButton.menuButton_, 'mouseenter', () => {
+          const playbackRate = this.player.playbackRate()
+          const rates = this.player.playbackRates()
+
+          // iterate through the items in reverse order as the highest is displayed first
+          let index = 0
+          for (let i = rates.length - 1; i >= 0; i--) {
```

Comment:
maybe a `.reverse()` combine with a `for of` loop would be more consciece

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reverse

Reply:
+1

OP please comment once code change made
Coz we won't be notified for new pushes

Reply:
Hold up
`reverse` update the array in place